src/Syncer/Command/SyncTimings.php

```diff
@@ -65,13 +72,18 @@ public function __construct(
         ReportsClient $reportsClient,
         InvoiceNinjaClient $invoiceNinjaClient,
         $clients,
-        $projects
+        $projects,
+        String $storageDir,
+        String $storageFileName
     ) {
         $this->togglClient = $togglClient;
         $this->reportsClient = $reportsClient;
         $this->invoiceNinjaClient = $invoiceNinjaClient;
         $this->clients = $clients;
         $this->projects = $projects;
+        $this->storageDir = $storageDir;
+        $this->storageFileName = $storageFileName;
```

Comment:
`$this->storageDir` and `$this->storageFileName` do not have properties in the class.